code/workspaces/infrastructure-api/src/dataaccess/projectsRepository.js

```diff
@@ -0,0 +1,40 @@
+import database from '../config/database';
+
+const Project = () => database.getModel('Project');
+
+async function getAll() {
+  return Project().find().exec();
+}
+
+async function getByKey(key) {
+  return Project().findOne({ projectKey: key }).exec();
+}
+
+async function exists(document) {
```

Comment:
Is this clearer if you make `projectKey` the function parameter then the line below is
```
return Project().exists({ projectKey });
```

Reply:
Agreed, I'll go through and update the names